include/fi_mem.h

```diff
@@ -104,7 +104,7 @@ static inline struct name * name ## _create(size_t size)	\
 {								\
 	struct name *fs;					\
 	fs = calloc(1, sizeof(*fs) + sizeof(entrytype) *	\
-			(roundup_power_of_two(size) - 1));	\
+		    (roundup_power_of_two(size)));		\
```

Comment:
Ah - cirq was the one I was actually thinking about.  As it is, I don't think anything in tree uses freestack yet (shm will).  So, I think we're good.  :)